srcalc/src/main/java/gov/va/med/srcalc/domain/variable/NumericalVariable.java

```diff
@@ -54,6 +59,58 @@ public void setMaxValue(final float maxValue)
         this.fMaxValue = maxValue;
     }
     
+    /**
+     * The units (if any) for the number. May be an empty string if units are
+     * not applicable, but will never be null.
+     * @return
+     */
+    @Basic
+    @Column(nullable = false, length = UNITS_MAX)
+    public String getUnits()
+    {
+        return fUnits;
+    }
+
+    /**
+     * Sets the units.
+     * @param units must not be null
+     * @throws NullPointerException if the given value is null
+     * @throws IllegalArgumentException if the given value is over 40 characters
+     */
+    public void setUnits(final String units)
```

Comment:
What would cause the units to be null and should we be checking for that (substituting an empty string instead)?